.gitignore

```diff
@@ -1,5 +1,5 @@
 # Compiled JS
-build
+/built
```

Comment:
surprising things happen on Mac when you have both BUILD and build/
surprising in a bad way